nixos/modules/services/networking/zerotierone.nix

```diff
@@ -45,7 +60,18 @@ in
         chown -R root:root /var/lib/zerotier-one
       '' + (concatMapStrings (netId: ''
         touch "/var/lib/zerotier-one/networks.d/${netId}.conf"
-      '') cfg.joinNetworks);
+      '') cfg.joinNetworks) + ''
+        if [ -L "${localConfFilePath}" ]
+        then
+          rm ${localConfFilePath}
+        elif [ -f "${localConfFilePath}" ]
+        then
+          mv ${localConfFilePath} ${localConfFilePath}.bak
+        fi
+      '' + optionalString (cfg.localConf != null) ''
```

Comment:
If `cfg.localConf == null` then I don't think anything in /var/lib/zerotier-one should modified.

Reply:
Fixed